charts/k8s-monitoring/templates/agent_config/_traces.river.txt

```diff
@@ -6,16 +6,22 @@
 {{- if (or .Values.traces.receiver.grpc.enabled .Values.traces.receiver.http.enabled) }}
 otelcol.receiver.otlp "trace_receiver" {
 
+{{- if .Values.traces.receiver.disable_debug_metrics }}
```

Comment:
This looks correct for the oltp receiver but it also needs to be copied over to the zipkin receiver block as well.

Reply:
Gotcha.

Would we prefer to introduce 2 separate config items i.e.

```
.Values.traces.receiver. oltp .disable_debug_metrics
.Values.traces.receiver.zipkin.disable_debug_metrics
```

Reply:
@basvdl that's probably the most robust option I guess, to have individual disable per receiver. Go for it.